src/screens/PlayScreen/Play.tsx

```diff
@@ -1,69 +1,390 @@
-import { Image, Text, View } from 'react-native';
+import {
+  Image,
+  Modal,
+  Pressable,
+  Text,
+  TouchableWithoutFeedback,
+  View,
+} from 'react-native';
+import { Audio } from 'expo-av';
+import React, { useState } from 'react';
+import Svg, { Circle, Path, Rect } from 'react-native-svg';
+// import Clipboard from '@react-native-clipboard/clipboard';
```

Comment:
do we still need the clipboard comments? or is this for testing

Reply:
clipboard/audio share functionality included in next steps